semeio/jobs/spearman_correlation_job/job.py

```diff
@@ -4,7 +4,14 @@
 from scipy.cluster.hierarchy import fcluster, linkage
 
 
-def spearman_job(measured_data, threshold):
+def spearman_job(
+    measured_data,
+    threshold,
+    criterion="inconsistent",
```

Comment:
Perhaps shared information about default values should be accessible for all jobs?

Reply:
Yes, as @jondequinor also suggested, they should probably share schema and default values. I can either resolve that here or we can leave that for a follow up, smaler PR 🤷 Let me know what you prefer...

Reply:
make a follow-up issue, and we are golden